src/modules/getfoods.js

```diff
@@ -31,13 +31,18 @@ const displayData = (arr) => {
   });
 };
 
-const getAllData = async (url) => {
+const displayCounter = (foodChosen, count) => {
+  foodChosen.innerHTML = `${foodChosen.textContent} (${count})`;
+};
+
+const getAllData = async (url, foodChosen) => {
   const request = new Request(url);
   const response = await fetch(request);
   const responseJson = await response.json();
-  const responsInfo = responseJson.meals;
+  const responseInfo = responseJson.meals;
+  displayCounter(foodChosen, responseInfo.length);
```

Comment:
- [ ] You did a good job implementing `length` to get the value for the counter, sadly this is not a requirement for the counter. Kindly, don't forget that the counter function should look for specific DOM elements (e.g. for the comments counter it should look for comments) and make the counting based on what is actually displayed on the page. 🔥😊

Reply:
I understand but note that from the task, we've not implemented the comment counter yet, and this was a proxy with would easily be substituted with the counter element when implemented

Reply:
Yes @Shedrack-Sunday, but we have implemented the cards counter and it should be counted here. They ask us to do that :shark:. I just copied it from the task :blush:. The `e.g.` part is just an example.
![Captura de pantalla de 2022-10-19 07-44-32](https://user-images.githubusercontent.com/105117832/196694343-04fc08f6-3378-4e76-977d-0d091bf4e0c5.png)

Reply:
Exactly the point. The 42 you see on the screen is from the API. It was not manually written. Can you point which dom you mean to be displayed in the counter? Because the 42 marks there is from the API count😊

Reply:
I understand @Shedrack-Sunday , but I forgot to tell you that also is a requirement no to use the API for that 
![Captura de pantalla de 2022-10-19 07-55-15](https://user-images.githubusercontent.com/105117832/196696851-ecf0856c-fbc3-4bd0-94a2-b608fbd8085d.png)

Reply:
With DOM elements they mean that we have to count for example the `article` or `li` that are inside our DOM.